src/main/java/com/andreamazzarella/contact_manager/Contact.java

```diff
@@ -0,0 +1,27 @@
+package com.andreamazzarella.contact_manager;
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Contact {
+    private HashMap<String, String> contactInfo;
+
+    public Contact(HashMap<String, String> contactInfo) {
+        this.contactInfo = contactInfo;
+    }
+
+    public String getName() {
+        return contactInfo.get("name");
+    }
+
+    public boolean nameContains(String searchTerm) {
```

Comment:
I can't decide if `nameContains` belongs here. If there was more search abilities, would you want methods like `nameContains,  phoneNumberContains, postcodeContains`, etc. Maybe one to watch out.. these kind of classes like "Contact", "User", "Thing" can become huge with loads of functionality.. I think they should only have data and accessors to that data.